cdist/conf/type/__check_messages/gencode-remote

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+#
+# 2019 Ander Punnar (ander-at-kvlt-dot-ee)
+#
+# This file is part of cdist.
+#
+# cdist is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cdist is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with cdist. If not, see <http://www.gnu.org/licenses/>.
+#
+
+if grep -Eq \
+    "$( cat "$__object/parameter/pattern" )" \
+    "$__messages_in"
+then
+    cat "$__object/parameter/execute" \
+        | tee "$__messages_out"
```

Comment:
@4nd3r Should `tee` be optional? :thinking:

Reply:
no.

```
TEE(1)

NAME
       tee - read from standard input and write to standard output and files
```
of course, maybe we don't want output in messages...